notebooks/tuning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>A little thought on ordering - would order of Pipelines -&gt; Grid Search -&gt; Grid Search with Pipelines make sense?  Just thinking it is a bit odd that we have not introduced Pipelines yet in the talk and might be worth introducing that before Grid Search (as it is main purpose of the talk). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I thought pipelines will be part of your intro? :) At least that is what I see in the hackmd? Or should I take this part, would be also fine for me..</p>